engines/agi/agi.h

```diff
@@ -533,6 +530,19 @@ struct AgiGame {
 	}
 };
 
+struct AgiDiskVolume {
+	uint32 disk;
+	uint32 offset;
+
+	AgiDiskVolume() : disk(_EMPTY), offset(0) {}
+	AgiDiskVolume(uint32 d, uint32 o) : disk(d), offset(o) {}
+};
+
+enum A2DirVersion {
```

Comment:
It would be nice to add a comment providing a bit more info about these versions

Reply:
Good idea!